math/src/field/fields/fft_friendly/quartic_babybear.rs

```diff
@@ -0,0 +1,552 @@
+    ) -> <Degree4BabyBearExtensionField as IsField>::BaseType {
+        let c0 = FieldElement::from_raw(<Self as IsField>::mul(a, b[0].value()));
+        let c1 = FieldElement::from_raw(<Self as IsField>::mul(a, b[1].value()));
+        let c2 = FieldElement::from_raw(<Self as IsField>::mul(a, b[2].value()));
+        let c3 = FieldElement::from_raw(<Self as IsField>::mul(a, b[3].value()));
+
+        [c0, c1, c2, c3]
+    }
+
+    fn add(
+        a: &Self::BaseType,
+        b: &<Degree4BabyBearExtensionField as IsField>::BaseType,
+    ) -> <Degree4BabyBearExtensionField as IsField>::BaseType {
+        let c0 = FieldElement::from_raw(<Self as IsField>::add(a, b[0].value()));
+        let c1 = FieldElement::from_raw(*b[1].value());
+        let c2 = FieldElement::from_raw(*b[2].value());
+        let c3 = FieldElement::from_raw(*b[3].value());
+
+        [c0, c1, c2, c3]
+    }
+
+    fn div(
+        a: &Self::BaseType,
+        b: &<Degree4BabyBearExtensionField as IsField>::BaseType,
+    ) -> <Degree4BabyBearExtensionField as IsField>::BaseType {
+        let b_inv = Degree4BabyBearExtensionField::inv(b).unwrap();
+        <Self as IsSubFieldOf<Degree4BabyBearExtensionField>>::mul(a, &b_inv)
+    }
+
+    fn sub(
+        a: &Self::BaseType,
+        b: &<Degree4BabyBearExtensionField as IsField>::BaseType,
+    ) -> <Degree4BabyBearExtensionField as IsField>::BaseType {
+        let c0 = FieldElement::from_raw(<Self as IsField>::sub(a, b[0].value()));
+        let c1 = FieldElement::from_raw(<Self as IsField>::neg(b[1].value()));
+        let c2 = FieldElement::from_raw(<Self as IsField>::neg(b[2].value()));
+        let c3 = FieldElement::from_raw(<Self as IsField>::neg(b[3].value()));
+        [c0, c1, c2, c3]
+    }
+
+    fn embed(a: Self::BaseType) -> <Degree4BabyBearExtensionField as IsField>::BaseType {
+        [
+            FieldElement::from_raw(a),
+            FieldElement::zero(),
+            FieldElement::zero(),
+            FieldElement::zero(),
+        ]
+    }
+
+    #[cfg(feature = "alloc")]
+    fn to_subfield_vec(
+        b: <Degree4BabyBearExtensionField as IsField>::BaseType,
+    ) -> alloc::vec::Vec<Self::BaseType> {
+        b.into_iter().map(|x| x.to_raw()).collect()
+    }
+}
+
+#[cfg(feature = "lambdaworks-serde-binary")]
+impl ByteConversion for [FieldElement<Babybear31PrimeField>; 4] {
+    #[cfg(feature = "alloc")]
+    fn to_bytes_be(&self) -> alloc::vec::Vec<u8> {
+        unimplemented!()
+    }
+
+    #[cfg(feature = "alloc")]
+    fn to_bytes_le(&self) -> alloc::vec::Vec<u8> {
+        unimplemented!()
+    }
+
+    fn from_bytes_be(_bytes: &[u8]) -> Result<Self, crate::errors::ByteConversionError>
+    where
+        Self: Sized,
+    {
+        unimplemented!()
+    }
+
+    fn from_bytes_le(_bytes: &[u8]) -> Result<Self, crate::errors::ByteConversionError>
+    where
+        Self: Sized,
+    {
+        unimplemented!()
+    }
+}
```

Comment:
I think it's better to implement this methods before merging the PR.

Reply:
Implemented  [here](https://github.com/lambdaclass/lambdaworks/pull/942/commits/18def0a2a17ed6c9b50518fdd744625d2c39e850)